sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1999,40 +1996,34 @@ public void reassignTasksOfCase(Case caze, boolean forceReassignment) {
 
 	}
 
-	private float calculateCompleteness(Case caze) {
+	@Override
+	public void updateCompletenessTask() {
+		List<String> getCompletenessCheckCaseList = getCompletenessCheckNeededCaseList();
+
+		if (!getCompletenessCheckCaseList.isEmpty()) {
+			long timeStart = System.currentTimeMillis();
+			IterableHelper
+				.executeBatched(getCompletenessCheckCaseList, 100, caseCompletionBatch -> caseService.updateCompleteness(caseCompletionBatch));
```

Comment:
Batch size of 10 cases was requested, to drop unused entities earlier from JPA cache.

Reply:
changed